### ISSUE TITLE
HTML5 Fallback

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sverhagen/mp3-browser#53]
```
Actually, I love the plugin, I was just wandering is it possible to code in a 
HTML5 fallback for devices not supporting Flash.
I wouldn't mind helping with the code or sponsoring it.
```

Original issue reported on code.google.com by `mrqw...@gmail.com` on 5 Jan 2015 at 9:35

**Comment by davidascherG** [comment]
THE OPPOSITE is what we need!!! ALL the tablets and phones out there do NOT support Flash but DO support HTML5. However, Flash players (at this point) seem to be more flexible and sophisticated than the HTML5 element. It would be better to see Flash use if IT is available with a fallback to HTML5 rather than the other way around as proposed here.

**Comment by sverhagen** [comment]
Implemented (mere-adequately) in 0.4.1. Sorry for the delay :wink:
